app/controllers/subscriber_authentication_controller.rb

```diff
@@ -51,6 +51,10 @@ def process_govuk_account
     render plain: "This GOV.UK account does not have a verified email address."
   end
 
+  helper_method def govuk_account_auth_enabled?
```

Comment:
huh this is new syntax, I've not seen this before.

So is this equivalent to doing:
```
def govuk_account_auth_enabled?
...
end

helper_method: :govuk_account_auth_enabled?
```

In some ways I like the shortening, but it's a bit jarring first time I saw it...

Can this be a private method too?

Reply:
Yeah, `def` returns the name of the method as a symbol, which is why this works.  I've seen both approaches in GOV.UK code.

I've made these private now.